frontend/src/pages/_app.js

```diff
@@ -1,5 +1,8 @@
 import { useState } from "react";
 import Head from "next/head";
+import './styles/nprogress.css';
+import NProgress from "nprogress";
+import Router from "next/router";
```

Comment:
Can you sort those please? 
The import order should be react -> next -> external libaries -> custom -> modules -> styles.